external/webserver/http_client.h

```diff
@@ -73,7 +75,8 @@ int http_parse_message(char *buf, int buf_len, int *method, char *url,
 					   struct http_keyvalue_list_t *params,
 					   struct http_client_t *client,
 					   struct http_client_response_t *response,
-					   struct http_req_message *req);
+					   struct http_req_message *req,
```

Comment:
how about to add description of the function because it has too many parameters.
I think description will help to understand the function

Reply:
done